src/content_script.js

```diff
@@ -2,7 +2,10 @@ var headerBar;
 
 function initOnce () {
     // check if element exists yet
-    headerBar = document.querySelector('body > div > .Header');
+    // GitHub history: header, then header.Header, now header.Header-old
+    // Gist history: header, then .Header, now .Header-old
+    //headerBar = document.querySelector('body > div > header, body > div > .Header, body > div > .Header-old');
+    headerBar = document.querySelector('header, .Header, .Header-old');
```

Comment:
I went for a more generalised selector, in case they change or move the headers in future.

I'm not aware of any secondary `header` elements deeper on GitHub pages which we should avoid. Are there any?